Source/Core/Core/HW/DSPHLE/UCodes/UCodes.h

```diff
@@ -61,15 +61,50 @@ class UCodeInterface
 
   CMailHandler& m_mail_handler;
 
-  enum EDSP_Codes : u32
-  {
-    DSP_INIT = 0xDCD10000,
-    DSP_RESUME = 0xDCD10001,
-    DSP_YIELD = 0xDCD10002,
-    DSP_DONE = 0xDCD10003,
-    DSP_SYNC = 0xDCD10004,
-    DSP_FRAME_END = 0xDCD10005,
-  };
+  static constexpr u32 TASK_MAIL_MASK = 0xFFFF'0000;
+  // Task-management mails, used for uCode switching. The DSP sends mail with 0xDCD1 in the high
+  // word to the CPU to report its status. The CPU reacts in different ways for different mails.
+  // Also, Zelda uCode titles use a slightly different handler compared to AX uCode titles. Libogc's
+  // mail handler is based on the AX one.
+  static constexpr u32 TASK_MAIL_TO_CPU = 0xDCD1'0000;
+  // Triggers a callback. No response is sent.
+  static constexpr u32 DSP_INIT = TASK_MAIL_TO_CPU | 0x0000;
+  // Triggers a callback. No response is sent.
+  static constexpr u32 DSP_RESUME = TASK_MAIL_TO_CPU | 0x0001;
+  // If the current task is canceled by the CPU, treated as DSP_DONE instead. Otherwise, this is
```

Comment:
Did you mean `treat it as DSP_NONE instead`? Not sure how to parse this otherwise.

Reply:
`[it is] treated [by the CPU] as [if it were] DSP_DONE instead` is what I had in mind. I'll expand it.